src/libp2p/connection/established/substream.rs

```diff
@@ -228,7 +228,7 @@ where
     /// A [`Event::InboundError`] can also be generated, either before or after the
     /// [`Event::InboundNegotiated`], but always before any [`Event::NotificationsInOpen`].
     ///
-    /// If [̀ InboundTy::Notifications`] is passed, then a [`Event::NotificationsInOpen`] will be
+    /// If [̀`InboundTy::Notifications`] is passed, then a [`Event::NotificationsInOpen`] will be
```

Comment:
```suggestion
    /// If [`InboundTy::Notifications`] is passed, then a [`Event::NotificationsInOpen`] will be
```

Sorry, I have some sort of weird shortcuts that sometimes make me accidentally add accents instead of backquotes.